{"cvrfdoc":{"document_title":"Red Hat Security Advisory: mod_python security update","document_distribution":"Copyright Â© 2021 Red Hat, Inc. All rights reserved.","document_references":{"reference":[{"description":"https://access.redhat.com/errata/RHSA-2004:058","type":"Self","url":"https://access.redhat.com/errata/RHSA-2004:058"},{"description":"http://www.modpython.org/pipermail/mod_python/2003-November/014532.html","type":"External","url":"http://www.modpython.org/pipermail/mod_python/2003-November/014532.html"}]},"aggregate_severity":"Low","document_tracking":{"initial_release_date":"2004-02-26T20:36:00Z","identification":{"id":"RHSA-2004:058"},"revision_history":{"revision":{"date":"2004-02-26T00:00:00Z","number":"1","description":"Current version"}},"generator":{"date":"2021-08-23T04:27:00Z","engine":"Red Hat SDEngine 3.0.0"},"current_release_date":"2004-02-26T00:00:00Z","version":"1","status":"Final"},"product_tree":{"relationship":[{"relates_to_product_reference":"3AS","product_reference":"mod_python-0:3.0.3-3.ent","full_product_name":{"product_id":"3AS:mod_python-0:3.0.3-3.ent","cpe":"cpe:/o:redhat:enterprise_linux:3::as","product_name":"mod_python-0:3.0.3-3.ent as a component of Red Hat Enterprise Linux AS version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"3Desktop","product_reference":"mod_python-0:3.0.3-3.ent","full_product_name":{"product_id":"3Desktop:mod_python-0:3.0.3-3.ent","cpe":"cpe:/o:redhat:enterprise_linux:3::desktop","product_name":"mod_python-0:3.0.3-3.ent as a component of Red Hat Desktop version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"3ES","product_reference":"mod_python-0:3.0.3-3.ent","full_product_name":{"product_id":"3ES:mod_python-0:3.0.3-3.ent","cpe":"cpe:/o:redhat:enterprise_linux:3::es","product_name":"mod_python-0:3.0.3-3.ent as a component of Red Hat Enterprise Linux ES version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"3WS","product_reference":"mod_python-0:3.0.3-3.ent","full_product_name":{"product_id":"3WS:mod_python-0:3.0.3-3.ent","cpe":"cpe:/o:redhat:enterprise_linux:3::ws","product_name":"mod_python-0:3.0.3-3.ent as a component of Red Hat Enterprise Linux WS version 3"},"relation_type":"Default Component Of"}],"branch":[{"name":"Red Hat Enterprise Linux","type":"Product Family","branch":[{"name":"Red Hat Enterprise Linux AS version 3","full_product_name":{"product_id":"3AS","cpe":"cpe:/o:redhat:enterprise_linux:3::as","product_name":"Red Hat Enterprise Linux AS version 3"},"type":"Product Name"},{"name":"Red Hat Desktop version 3","full_product_name":{"product_id":"3Desktop","cpe":"cpe:/o:redhat:enterprise_linux:3::desktop","product_name":"Red Hat Desktop version 3"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux ES version 3","full_product_name":{"product_id":"3ES","cpe":"cpe:/o:redhat:enterprise_linux:3::es","product_name":"Red Hat Enterprise Linux ES version 3"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux WS version 3","full_product_name":{"product_id":"3WS","cpe":"cpe:/o:redhat:enterprise_linux:3::ws","product_name":"Red Hat Enterprise Linux WS version 3"},"type":"Product Name"}]},{"name":"mod_python-0:3.0.3-3.ent","full_product_name":{"product_id":"mod_python-0:3.0.3-3.ent","cpe":"cpe:/o:redhat:enterprise_linux:3::as","product_name":"mod_python-3.0.3-3.ent.src.rpm"},"type":"Product Version"}]},"document_publisher":{"issuing_authority":"Red Hat Product Security","contact_details":"secalert@redhat.com","type":"Vendor"},"vulnerability":[{"notes":{"note":"security flaw"},"cve":"CVE-2003-0973","references":{"reference":[{"description":"CVE-2003-0973","url":"https://access.redhat.com/security/cve/CVE-2003-0973"},{"description":"bz#1617112: CVE-2003-0973 security flaw","url":"https://bugzilla.redhat.com/show_bug.cgi?id=1617112"}]},"release_date":"2003-11-28T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["3AS:mod_python-0:3.0.3-3.ent","3Desktop:mod_python-0:3.0.3-3.ent","3ES:mod_python-0:3.0.3-3.ent","3WS:mod_python-0:3.0.3-3.ent"],"type":"Fixed"}},"remediations":{"remediation":{"description":"Before applying this update, make sure all previously released errata\nrelevant to your system have been applied.\nTo update all RPMs for your particular architecture, run:\nrpm -Fvh [filenames]\nwhere [filenames] is a list of the RPMs you wish to upgrade.  Only those\nRPMs which are currently installed will be updated.  Those RPMs which are\nnot installed but included in the list will not be updated.  Note that you\ncan also use wildcards (*.rpm) if your current directory *only* contains the\ndesired RPMs.\nPlease note that this update is also available via Red Hat Network.  Many\npeople find this an easier way to apply updates.  To use Red Hat Network,\nlaunch the Red Hat Update Agent with the following command:\nup2date\nThis will start an interactive process that will result in the appropriate\nRPMs being upgraded on your system.\nIf up2date fails to connect to Red Hat Network due to SSL\nCertificate Errors, you need to install a version of the\nup2date client with an updated certificate.  The latest version of\nup2date is available from the Red Hat FTP site and may also be\ndownloaded directly from the RHN website:\nhttps://rhn.redhat.com/help/latest-up2date.pxt","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2004:058"}},"threats":{"threat":{"description":"Low","type":"Impact"}},"ordinal":"1"},{"notes":{"note":"mod_python remote DoS"},"cve":"CVE-2004-0096","references":{"reference":[{"description":"CVE-2004-0096","url":"https://access.redhat.com/security/cve/CVE-2004-0096"},{"description":"bz#430510: CVE-2004-0096 mod_python remote DoS","url":"https://bugzilla.redhat.com/show_bug.cgi?id=430510"}]},"release_date":"2004-01-22T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["3AS:mod_python-0:3.0.3-3.ent","3Desktop:mod_python-0:3.0.3-3.ent","3ES:mod_python-0:3.0.3-3.ent","3WS:mod_python-0:3.0.3-3.ent"],"type":"Fixed"}},"remediations":{"remediation":{"description":"Before applying this update, make sure all previously released errata\nrelevant to your system have been applied.\nTo update all RPMs for your particular architecture, run:\nrpm -Fvh [filenames]\nwhere [filenames] is a list of the RPMs you wish to upgrade.  Only those\nRPMs which are currently installed will be updated.  Those RPMs which are\nnot installed but included in the list will not be updated.  Note that you\ncan also use wildcards (*.rpm) if your current directory *only* contains the\ndesired RPMs.\nPlease note that this update is also available via Red Hat Network.  Many\npeople find this an easier way to apply updates.  To use Red Hat Network,\nlaunch the Red Hat Update Agent with the following command:\nup2date\nThis will start an interactive process that will result in the appropriate\nRPMs being upgraded on your system.\nIf up2date fails to connect to Red Hat Network due to SSL\nCertificate Errors, you need to install a version of the\nup2date client with an updated certificate.  The latest version of\nup2date is available from the Red Hat FTP site and may also be\ndownloaded directly from the RHN website:\nhttps://rhn.redhat.com/help/latest-up2date.pxt","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2004:058"}},"threats":{"threat":{"description":"Moderate","type":"Impact"}},"discovery_date":"2004-01-22T00:00:00Z","ordinal":"2"}],"document_notes":{"note":["Updated mod_python packages that fix a denial of service vulnerability are\nnow available for Red Hat Enterprise Linux.","mod_python embeds the Python language interpreter within the Apache httpd\nserver.\nA bug has been found in mod_python versions 2.7.10 and earlier that can\nlead to a denial of service vulnerability.  The Common Vulnerabilities and\nExposures project (cve.mitre.org) has assigned the name CAN-2003-0973 to\nthis issue.\nAlthough Red Hat Enterprise Linux shipped with a version of mod_python that\ncontains this bug, our testing was unable to trigger the denial of service\nvulnerability.  However, mod_python users are advised to upgrade to these\nerrata packages, which contain a backported patch that corrects this bug.","This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original."]},"document_type":"Security Advisory"}}